sdk/lib/compartment_helpers/README.md

```diff
@@ -4,5 +4,4 @@ Compartment helpers library
 This library includes functions that help securing compartment boundaries.
 
 - [`claim_fast.cc`] contains the `heap_claim_fast` function.
-
-
+- [`check_pointer.cc`] contains the `::check_pointer` function.
```

Comment:
```suggestion
- [`check_pointer.cc`] contains the `check_pointer` function.
```

We don't use :: in the document for anything else in the global namespace in these libraries.